apps/show/helpers/view_helpers.coffee

```diff
@@ -269,3 +276,19 @@ module.exports =
       artistText
       info
     ]).join(' ')
+
+  bestAddress: (location) ->
+    location.display or @displayAddress(location)
```

Comment:
The location returned from Gravity isn't really consistent across models. Let's wrap it in a helper here where we can keep tweaking, if necessary.